starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,52 @@
 // Names and Input
+var hacker1 = "Bob";
+
+console.log("The drivers name is " + hacker1);
+var hacker2 = prompt("What is the navigators name?");
+console.log("The navigators name is " + hacker2);
 
 
 //Conditionals
 
+function longestName(){
+  var driverLength = hacker1.length;
+  var navigatorLength = hacker2.length;
+  if (driverLength > navigatorLength){
+    console.log("The Driver has the longest name, it has " + driverLength + " characters");
+  } else if (driverLength < navigatorLength) {
+    console.log("Yo, navigator got the longest name, it has " +  navigatorLength  + "  characters");
+  } else {
+    console.log("wow, you both got equally long names, " + navigatorLength +  "characters!!");
+  }
+}
+
+function driverLettersCaps(){
```

Comment:
The assignment here is to print all the characters of the driver's name, separated by a space and in capitals, so we have to do something like: variable.split().join().toUpperCase(). This is just a hint. What would be a solution?